ibis/expr/api.py

```diff
@@ -152,6 +152,14 @@ def desc(expr):
 def timestamp(value):
     """
     Returns a timestamp literal if value is likely coercible to a timestamp
+
+    Parameters
+    ----------
+    value : timestamp value as string
+
+    Examples
+    --------
+    result : Timestamp scalar
```

Comment:
This actually returns an `ibis.expr.types.TimestampScalar`, you can just write `TimestampScalar`.